benchopt/plotting/html/templates/result.mako.html

```diff
@@ -92,6 +92,16 @@ <h1 class="ml11">
                                 <span class="slider round"></span>
                             </label>
                         </div>
+                        <div id="xaxis-type-form-group" class="mt-8 px-8">
+                            <label for="change_scaling" class="block text-sm font-medium text-white">X axis type</label>
+                            <div class="mt-1 rounded-md shadow-sm">
+                                <select id="change_xaxis_type" class="focus:ring-blue-500 focus:border-blue-500 block w-full h-8 px-4 sm:text-sm border-gray-300 rounded-md" onchange="setState({xaxis_type: this.value})">
+                                    <option value="time">Time</option>
+                                    <option value="iteration">Iteration</option>
+                                    <option value="tolerance">Tolerance</option>
```

Comment:
can this be parametrized to avoid putting `iteration` or `tolerance` if it does not exists in the data?
Actually, all these values should come from the `['time'] + list(df['stopping_strategy'].unique())`

Reply:
I opted for dynamically creating the options depending on the selected dataset and objective columns. In fact, some solvers might be skipped in some cases.

Any thoughts?